bin/rabbit_plot_pulls_and_impacts.py

```diff
@@ -248,28 +251,29 @@ def make_bar(
                 col=1,
             )
 
-        fig.add_trace(
-            make_bar(
-                key="impact_down",
-                name="-1σ impact",
-                color="#e41a1c",
-                text_on_bars=text_on_bars,
-                opacity=0.5 if include_ref else 1,
-            ),
-            row=1,
-            col=1,
-        )
-        if include_ref:
+        if not group:
```

Comment:
Why do you exclude those for grouped impacts? In principle grouped impacts could be asymmetric as well in case of non-gaussian uncertainties. Now it looks like this:
Your PR:
https://cmsmwbot.web.cern.ch/Rabbit/PRValidation/PR71/2025_10_08/impacts_grouped_sig.png
Reference:
https://cmsmwbot.web.cern.ch/Rabbit/PRValidation/ReferenceRuns/2025_10_06_2890f26/impacts_grouped_sig.png
If one wants one-sided impacts there is an option.
I suggest reverting this change if there is not reason.

Reply:
I was more targeting the use case for (I think) the max group case, where we show the abs impact.